### ISSUE TITLE
Add qr error correction option

### ISSUE THREAD

**Issue by andersmmg** [vnphanquang/svelte-put#287]
I'd like an option to change the error correction level on the qr module

**Comment by vnphanquang** [comment]
Now available in [@svelte-put/qr@1.1.0](https://github.com/vnphanquang/svelte-put/releases/tag/%40svelte-put%2Fqr%401.1.0) via the `errorCorrectionLevel`. Closing for now but please feel free to reopen if you find any issue.

Cheers!

**Comment by davjhan** [comment]
FYI, I'm on `2.1.1`, and while the docs say:

```
	/**
	 * @deprecated use `correction` instead
	 */
	errorCorrectionLevel?: QRCodeOptions['correction'];
};
```

`correction` doesn't do anything (doesn't change the generated output) meanwhile `errorCorrectionLevel` does.

**Comment by vnphanquang** [comment]
Thanks, I must have missed it during the latest release. Apologies. Will look into it shortly

**Comment by vnphanquang** [comment]
> FYI, I'm on `2.1.1`, and while the docs say:
> [...]
> `correction` doesn't do anything (doesn't change the generated output) meanwhile `errorCorrectionLevel` does.

The fix is now available in [@svelte-put/qr@2.1.2](https://github.com/vnphanquang/svelte-put/releases/tag/%40svelte-put%2Fqr%402.1.2). Apologies for the delay; life got in the way 😄.
